wordpress-mobile/gutenberg-mobile
# Use fix for RichTextInputEvent

Gutenberg side fix to effectively bypass the `RichTextInputEvent` component.

To test:

* Run the demo app
* Tap inside a paragraph block
* Tap some characters. The should work as normal without crashing.
